scripts/test/convertMinus500DataPointsToNull.test.js

```
@@ -0,0 +1,35 @@
+const mongo = require('mongodb-lib')
+
+const convert500toNull =  require('../convertMinus500DataPointsToNull')
+const { dataPoints } = require('./seeds/dataPoints')
+
+describe('migrate dataPoints collection', () => {
+
+  let db
+  let dataPointsDB
+
+  beforeAll(async () => {
+    db = await mongo.db()
+    dataPointsDB = await db.collection('dataPoints')
+  })
+
+  afterAll(async () => {
+    await mongo.conn().close()
+  })
+
+  beforeEach(async ()=>{
+    await dataPointsDB.insertMany(dataPoints)
```

see comment on seed file. 
should you expand these tests, this pattern locks us into asserting on default data on the seed that is obscured in another file. 
If the seed instead becomes a `generateSeed`, we can explicitly pass in the dataPoint values we would like to test against in the tests, which becomes really convenient